apps/plotting.py

```diff
@@ -1541,6 +1550,10 @@ def extract_cutout(object_data, time0, kind):
 
     cutout = readstamp(r.content, gzipped=False)
 
+    if pdf_['i:isdiffpos'].values[position] == 'f' and kind == 'difference':
```

Comment:
`i:isdiffpos` is not included in `object_data` when creating `pdf_`. This should be declared in `draw_cutouts_quickview` when defining columns (`cols = `).